subjects/returna/README.md

```diff
@@ -0,0 +1,20 @@
+## returna
+
+### Instructions
+
+Write a program that takes a `string` as an argument, if the string contains the letter `"a"` then print `"Contains the letter"` followed by a newline.
```

Comment:
Instruction fix: "...if the `string` contains the letter `a` then print `Contains the letter` followed by a newline `\n`."

Reply:
fixed